src/sasctl/pzmm/pickleModel.py

```diff
@@ -3,14 +3,14 @@
 
 
 # %%
-import os
+from pathlib import Path
 
 import pickle
 
 # %%
 class PickleModel():
     
-    def pickleTrainedModel(self, trainedModel, modelPrefix, pPath=os.getcwd()):
+    def pickleTrainedModel(trainedModel, modelPrefix, pPath=Path.cwd()):
```

Comment:
Change to
    def pickleTrainedModel(self, trainedModel, modelPrefix, pPath=Path.cwd()):